devtools/create_project/create_project.cpp

```diff
@@ -1147,6 +1171,63 @@ ToolList getAllTools() {
 	return tools;
 }
 
+MSVCList getAllMSVCVersions() {
+	const size_t msvcCount = sizeof(s_msvc) / sizeof(s_msvc[0]);
+
+	MSVCList msvcVersions;
+	for (size_t i = 0; i < msvcCount; ++i)
+		msvcVersions.push_back(s_msvc[i]);
+
+	return msvcVersions;
+}
+
+const MSVCVersion *getMSVCVersion(int version) {
+	const size_t msvcCount = sizeof(s_msvc) / sizeof(s_msvc[0]);
+
+	for (size_t i = 0; i < msvcCount; ++i) {
+		if (s_msvc[i].version == version)
+			return &s_msvc[i];
+	}
+
+	return NULL;
+}
+
+int getInstalledMSVC() {
+	int latest = 0;
+#if defined(_WIN32) || defined(WIN32)
```

Comment:
Is this required?

Reply:
create_project can be compiled on non-Windows platforms (eg. to generate Xcode projects).